client/rest-high-level/src/test/java/org/elasticsearch/client/SnapshotIT.java

```diff
@@ -270,4 +290,28 @@ public void testDeleteSnapshot() throws IOException {
 
         assertTrue(response.isAcknowledged());
     }
+
+    private static Map<String, Object> randomUserMetadata() {
+//        if (randomBoolean()) {
```

Comment:
Remove it?

Reply:
Ah, whoops, I commented this out to get a higher percentage of test runs to verify that the rest of this works properly. I'll uncomment.